framework/src/org/checkerframework/framework/type/AnnotationClassLoader.java

```diff
@@ -89,10 +86,11 @@
     private final URL resourceURL;
 
     /**
-     * The loaded annotation classes. Call {@link #getLoadedAnnotationClasses} rather than using
-     * this field directly as it may be null.
+     * The loaded annotation classes, which includes bunded and external annotations that are deemed
+     * supported by each checker. Call {@link #getBundledAnnotationClasses()} rather than using this
```

Comment:
What does the `each` quantify over? Should it simply be `the`?

Reply:
Fixed